### PR TITLE
Add weblab docker to our monorepo

### LINKED ISSUE #143: Andy tests fail on Windows
Running tests on Windows results in some failing tests. This occurs on GitHub actions, I have not tested it on an actual machine.

Could possibly be related to security manager issues.

This does not affect running tests via Maven: the Maven plugin works fine on Windows.

---

> It seems like some of our tests fail on Windows for some reason. But the weird part is that a few unit tests fail, but none of our system tests do. All of those unit tests just test basic features (not edge cases), so that should in theory also cause system tests to fail... :thinking:

_Originally posted by @martinmladenov in https://github.com/cse1110/andy/issues/142#issuecomment-1289016956_

### LINKED ISSUE #172: Move weblab docker repository to this one
So we just have one simple repository with everything.

### LINKED ISSUE #184: Fix solution for summer and unique PBT exercises
Now that we have trials=100, both of these test suites are now unstable. We gotta fix them ASAP!

These two are currently excluded from the CI test suite. See `AllAssignmentsTest`. We need to enable them again once we fix them.

### PR DESCRIPTION
**Pull request by mauricioaniche**
Closes #172 

Closes #184 

Closes #143

### CODE REVIEW

File: weblab-docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    bash \
+    sudo \
+    wget \
+    unzip \
+ && rm -rf /var/lib/apt/lists/*
+
+SHELL ["/bin/bash", "-c"]
+
+# Download the andy code from the remote git repository
+RUN git clone --depth 1 https://github.com/cse1110/andy.git
```
[the remark this answers is not in the document]
Comment (mauricioaniche) [reply weblab-docker/Dockerfile:20]: Great idea. Let me try!

File: .github/workflows/assignments_test_docker.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        working-directory: ./weblab-docker
+        run: |
+          sed -i 's/gtar/tar/' Makefile.inc
+
+      - name: Build Docker image
+        working-directory: ./weblab-docker
+        run: |
+          make
+
+      - name: Execute python script
```
Comment (mauricioaniche) [inline .github/workflows/assignments_test_docker.yml:26]: The name here can explain what this script does instead of "execute script"

Can we also make this script not to run all the assignments? IIRC, this would take around 20 minutes. Can we just randomly select one or two per folder?
